cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,39 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+let user;
+let article;
+describe('Conduit', () => {
+  beforeEach(() => {
+    cy.visit('');
```

Comment:
It's a good practice to use absolute URLs in `cy.visit()`. Instead of an empty string, you should provide the base URL of your application.